training/distributed_training/pytorch/model_parallel_v2/mixtral/smp-train-mixtral-fsdp-ep-fp8.ipynb

```diff
@@ -0,0 +1,1043 @@
+    "            \"num_layers\": 80,\n",
+    "        },\n",
+    "    },\n",
+    "    \"llama_v2\": {\n",
+    "        7: {\n",
+    "            \"hidden_width\": 4096,\n",
+    "            \"llama_intermediate_size\": 11008,\n",
+    "            \"max_context_width\": 2048,\n",
+    "            \"num_heads\": 32,\n",
+    "            \"num_layers\": 32,\n",
+    "        },\n",
+    "        65: {\n",
+    "            \"hidden_width\": 8192,\n",
+    "            \"llama_intermediate_size\": 22016,\n",
+    "            \"max_context_width\": 2048,\n",
+    "            \"num_heads\": 64,\n",
+    "            \"num_layers\": 80,\n",
+    "        },\n",
+    "    },\n",
+    "    \"mixtral\": {\n",
+    "        \"small\": {\n",
+    "            \"hidden_width\": 2048,\n",
+    "            \"max_context_width\": 2048,\n",
+    "            \"num_heads\": 16,\n",
+    "            \"num_layers\": 8,\n",
+    "            \"num_local_experts\": 4,\n",
+    "        },\n",
+    "        \"large\": {\n",
+    "            \"hidden_width\": 4096,\n",
+    "            \"max_context_width\": 2048,\n",
+    "            \"num_heads\": 32,\n",
+    "            \"num_layers\": 32,\n",
+    "            \"num_local_experts\": 8,\n",
+    "        },\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "model_params = model_configs.get(model_type, {}).get(model_size)\n",
+    "if model_params is None:\n",
+    "    raise RuntimeError(\n",
+    "        f\"Unknown model config from (name, size) = ({model_type}, {model_size:02d}B).\"\n",
+    "    )\n",
+    "\n",
+    "hyperparameters.update(model_params)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Specify Essential Parameters for a SageMaker Training Job\n",
+    "\n",
+    "Next, you use the `SageMaker Estimator class` to define a SageMaker Training Job,\n",
+    "passing values through the following parameters for training job name,\n",
+    "the number of EC2 instances, the instance type,\n",
+    "and the size of the volume attached to the instances.\n",
+    "\n",
+    "- `instance_count`\n",
+    "- `instance_type`\n",
+    "- `volume_size`\n",
+    "- `base_job_name`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Update the Type and Number of EC2 Instance to Use\n",
+    "\n",
+    "The instance type and the number of instances you specify to the `instance_type` and `instance_count` parameters,\n",
+    "respectively, determine the total number of GPUs (`world_size`).\n",
+    "\n",
+    "$$\\text{world_size = (the number of GPUs on a single instance)} \\times \\text{(the number of instances)}$$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# @title Setup instances: type and count.\n",
+    "\n",
+    "instance_type = \"ml.p5.48xlarge\"\n",
+    "\n",
+    "instance_count = 8\n",
```

Comment:
We don't need 8 p5 nodes to run with this model config, 1 or 2 should be enough

Reply:
Updated!